packages/ipfs/docs/FAQ.md

```diff
@@ -13,7 +15,26 @@
 
 There is DHT support for js-IPFS in the form of [libp2p/js-libp2p-kad-dht](https://github.com/libp2p/js-libp2p-kad-dht) but it is not finished yet, and may not be the right solution to the problem.
 
-Historically js-IPFS has targeted browser environments.  The constraints imposed by browsers do not typically make for good DHT participants - people do not tend to stay on a page for long enough to make or answer DHT queries, and even if they did, most nodes on the network talk TCP - the browser can neither open TCP ports on remote hosts nor accept TCP connections.
+### Node.js
+
+To enable DHT support, before starting your daemon run:
+
+```console
+$ jsipfs config Routing.Type dht
+```
+
+The possible values for `Routing.Type` are:
+
+ -  `'none'` the default, this means the DHT is turned off any you must manually dial other nodes
+ -  `'dht'` start the node in DHT client mode, if it works out it is dialable from outside the network, upgrade to server mode
```

Comment:
```suggestion
 -  `'dht'` start the node in DHT client mode, if it is discovered to be publicly dialable it will automatically switch to server mode
```